platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/validation/DefaultInternalEventValidator.java

```diff
@@ -157,16 +171,57 @@ public DefaultInternalEventValidator(
      * @return true if the required fields of the event are non-null, otherwise false
      */
     private boolean areRequiredFieldsNonNull(@NonNull final PlatformEvent event) {
```

Comment:
It wasn't already in this class, but should we also be checking `maxTransactionCountPerEvent` and `transactionMaxBytes` here? 

`maxTransactionCountPerEvent` currently isn't being validated *anywhere*, and `transactionMaxBytes` isn't checked anywhere in intake (it's only checked on the node submitting the transaction)

Reply:
I will create a separate ticket for this